src/De/DeJson.jl

```diff
@@ -2,15 +2,326 @@ module DeJson
 
 export deser_json
 
-using ..ParJson
+using YYJSON
+import ..Serde
+import ..Serde: deser, eldeser, isempty, custom_name, default_value, nulltype
+import ..Serde: WrongType, CustomType, NullType, PrimitiveType, ArrayType, DictType, NTupleType
 import ..to_deser
 
+# Any
+
+function deser(::Type{Any}, val_ptr::Ptr{YYJSONVal})
+    return if yyjson_is_str(val_ptr)
+        unsafe_string(yyjson_get_str(val_ptr))
+    elseif yyjson_is_raw(val_ptr)
+        unsafe_string(yyjson_get_raw(val_ptr))
+    elseif yyjson_is_real(val_ptr)
+        yyjson_get_real(val_ptr)
+    elseif yyjson_is_int(val_ptr)
+        yyjson_get_int(val_ptr)
+    elseif yyjson_is_bool(val_ptr)
+        yyjson_get_bool(val_ptr)
+    elseif yyjson_is_obj(val_ptr)
+        deser(Dict{String,Any}, val_ptr)
+    elseif yyjson_is_arr(val_ptr)
+        deser(Vector{Any}, val_ptr)
+    end
+end
```

Comment:
```suggestion
function deser(::Type{Any}, value_ptr::Ptr{YYJSONVal})
    return if yyjson_is_str(value_ptr)
        unsafe_string(yyjson_get_str(value_ptr))
    elseif yyjson_is_raw(value_ptr)
        unsafe_string(yyjson_get_raw(value_ptr))
    elseif yyjson_is_real(value_ptr)
        yyjson_get_real(value_ptr)
    elseif yyjson_is_int(value_ptr)
        yyjson_get_int(value_ptr)
    elseif yyjson_is_bool(value_ptr)
        yyjson_get_bool(value_ptr)
    elseif yyjson_is_obj(value_ptr)
        deser(Dict{String,Any}, value_ptr)
    elseif yyjson_is_arr(value_ptr)
        deser(Vector{Any}, value_ptr)
    else
        throw(WrongType("Unsupported JSON type for `Any`."))
    end
end
```

Reply:
I didn't add an error throw here because it happens outside